### PR TITLE
change emoji for:  notifying new kubernetes version is available

### LINKED ISSUE #7792: change emoji for telling that new kubernetes is available
```
make: 'out/minikube' is up to date.
😄  minikube v1.9.2 on Ubuntu 20.04
✨  Using the docker driver based on user configuration
👍  Kubernetes 1.18.0 is now available. If you would like to upgrade, specify: --kubernetes-version=1.18.0
👍  Starting control plane node minikube in cluster minikube
🔥  Creating docker container (CPUs=2, Memory=3900MB) ..
```

### PR DESCRIPTION
**Pull request by radeksm**
Change emoji for telling that new kubernetes is available.

fixes #7792

### CODE REVIEW

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -997,7 +997,9 @@ func getKubernetesVersion(old *config.ClusterConfig) string {
 
 	}
 	if defaultVersion.GT(nvs) {
-		out.T(out.ThumbsUp, "Kubernetes {{.new}} is now available. If you would like to upgrade, specify: --kubernetes-version={{.new}}", out.V{"new": defaultVersion})
+        out.T(out.New,
+              "Kubernetes {{.new}} is now available. If you would like to upgrade, specify: --kubernetes-version={{.new}}",
```
Comment (medyagh) [inline cmd/minikube/cmd/start.go:1001]: why there is a new line after ,  could we keep it in same line?

**Comment by medyagh** [comment]
@radeksm do you mind putting the output before and after this PR ?

File: cmd/minikube/cmd/start.go (hunk 1 above)
Comment (radeksm) [reply cmd/minikube/cmd/start.go]: Ohhh my old habits from OpenStack.

**Comment by radeksm** [comment]
> @radeksm do you mind putting the output before and after this PR ?



> @radeksm do you mind putting the output before and after this PR ?

Here is how it looks like:

[radek@c8k15 ~]$ ./minikube  --v=4 --driver=docker  --kubernetes-version=v1.17.0   start 
😄  minikube v1.10.0-beta.0 on Centos 8.1.1911
✨  Using the docker driver based on user configuration
🆕  Kubernetes 1.18.0 is now available. If you would like to upgrade, specify: --kubernetes-version=1.18.0
👍  Starting control plane node minikube in cluster minikube
💾  Downloading Kubernetes v1.17.0 preload ...
    > preloaded-images-k8s-v2-v1.17.0-docker-overlay2-amd64.tar.lz4: 539.92 MiB
🔥  Creating docker container (CPUs=2, Memory=1485MB) ...
🐳  Preparing Kubernetes v1.17.0 on Docker 19.03.2 ...

**Comment by radeksm** [comment]
Unitest failed but failure seems to be unrelated to my change:

`2020-04-22T19:55:02.3881542Z --- FAIL: TestNewSSHClient (0.08s)`
`2020-04-22T19:55:02.3881741Z     ssh_mock.go:176: Listening on 127.0.0.1:41431`
`2020-04-22T19:55:02.3881836Z     ssh_mock.go:83: Accepting...`
`2020-04-22T19:55:02.3881939Z     ssh_mock.go:83: Accepting...`
`2020-04-22T19:55:02.3882079Z     ssh_mock.go:135: exec request received: &{Type:exec WantReply:true Payload:[0 0 0 3 102 111 111] ch:0xc0001ee780 mux:<nil>}`
`2020-04-22T19:55:02.3882232Z     sshutil_test.go:67: Error running "foo": wait: remote command exited without exit status or exit signal`
`2020-04-22T19:55:02.3882357Z     ssh_mock.go:192: Stopping`
`2020-04-22T19:55:02.3882449Z panic: Fail in goroutine after TestNewSSHClient has completed`

**Pull request merged**